python/tvm/relay/frontend/pytorch.py

```diff
@@ -1755,7 +1765,8 @@ def pad(inputs, input_types):
 
         return pad
 
-    def clamp(self, inputs, input_types):
+    def clamp_common(self, inputs, min_only=False, max_only=False):
```

Comment:
I wonder if it makes sense to instead pass `None` as inputs, as currently done in the code (and in TorchScript) when you script `clamp`.

Reply:
Done.